modules/migration-upgrading-mtc-on-ocp-4.adoc

```diff
@@ -9,6 +9,20 @@
 
 You can upgrade the {mtc-full} ({mtc-short}) on {product-title} {product-version} by using the Operator Lifecycle Manager.
 
+[IMPORTANT]
+.Migration paths
+====
+* Migrating from {product-title} 3 to {product-title} 4 requires a legacy {mtc-short} Operator and {mtc-short} 1.7.x.
+* Migrating from {mtc-short} 1.7.x to {mtc-short} 1.8.x is not supported.
+* You must use {mtc-short} 1.7.x to migrate anything with a source of {product-title} 4.9 or earlier.
+** {mtc-short} 1.7.z must be used on both source and destination.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x may be used. However, but it must be the same MTC 1.Y.z on both source & destination.
+** Migration from source {mtc-short} 1.7.z to destination {mtc-short} 1.8.z is unsupported.
+** Migration from source {mtc-short} 1.8.z to destination {mtc-short} 1.7.z is unsupported.
+** Migration from source {mtc-short} 1.7.z to destination {mtc-short} 1.7.z is supported.
+** Migration from source {mtc-short} 1.8.z to destination {mtc-short} 1.8.z is supported
```

Comment:
Docs already use dot-x throughout.

```suggestion
** {mtc-short} 1.7.z must be used on both source and destination.
* MTC 1.8.x only supports migrations from {product-title} 4.10 or later to {product-title} 4.10 or later. For migrations only involving cluster versions 4.10 and later, either 1.7.x or 1.8.x may be used. However, it must be the same MTC version on both source & destination.
** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.8.x is unsupported.
** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.7.x is unsupported.
** Migration from source {mtc-short} 1.7.x to destination {mtc-short} 1.7.x is supported.
** Migration from source {mtc-short} 1.8.x to destination {mtc-short} 1.8.x is supported
```

For "_must be the same MTC 1.Y.z on both source & destination_", I think it'll be cleaner and with this suggestion, but rework that as necessary if it isn't sufficient.  :sunglasses: